dekalee/mailjet-bundle/0.5/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "Dekalee\\MailjetBundle\\DekaleeMailjetBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "DEKALEE_MAILJET_API_KEY": "your-api_key",
+        "DEKALEE_MAILJET_SECRET_KEY": "@@your-secret_key",
```

Comment:
Can you by default comment the variables and add the statements in the post-install.txt ?

Reply:
Why the double at-sign (@)?

Reply:
@Nyholm I saw it in another recipes.

I will remove it for a better clarity.